### PR TITLE
Prepare RA chrome for fractional DPI/UI scaling

### PR DESCRIPTION
**Pull request by pchote**
This PR is a sibling to #17494, which outlines the motivations and requirements, and gives some hints on how to efficiently test and review this.

RA is split into its own PR as it is a much larger rework than TD or the content installer.

The previous `chrome.png` is split into two new files, with `glyphs.png` generated from SVG at multiple resolutions and `sidebar.png` holding the native pixel-based data (with significantly reduced duplication).

The loadscreen has also been rebuilt and generated from SVG.

This also has to touch the music icon mappings in D2k and TS because of changes to the common widget layout.

Depends on #17489.
Artwork source are PRed in: https://github.com/OpenRA/ArtSrc/pull/8.

### CODE REVIEW

**Comment by pchote** [comment]
Updated to fix a tiling error with `dialog`.

**Comment by dragunoff** [comment]
Looking good. I see you've added active states for command icons but for some reason "deploy", "scatter" and "stop" don't flash the active icon (like in TD for example). It might even be better that way but I'm curious why it happens?

**Comment by pchote** [comment]
Good eye! The change to the active states weren't intentional - fixed.

**Comment by pchote** [comment]
Updated to remove the vestigial debug menu button from the sidebar, which was causing a crash when starting a game with the debug menu lobby option checked. We don't have vector artwork for this glyph, and the effort to create a new one from scratch isn't justified by saving a mouse click when most of us use the chat commands.

**Comment by pchote** [comment]
Rebased.

**Review by dragunoff — DISMISSED** [review]
Looks good in-game.

I think the RA sidebar and dialogs can be recreated in SVG without too much effort. In a future PR of course. I may look into that.

**Comment by pchote** [comment]
Rebased and updated with a glyph for #17549.

File: mods/ra/chrome.yaml [hunk 1]
```diff
@@ -317,83 +320,88 @@ dialog5:
...
+		kick: 153, 0, 11, 11
+		protected: 0, 17, 12, 13
+		protected-disabled: 17, 17, 12, 13
+		authentication: 34, 17, 12, 13
+		authentication-disabled: 51, 17, 12, 13
+		admin-registered: 0, 51, 16, 16
+		admin-anonymous: 34, 51, 16, 16
+		player-registered: 17, 51, 16, 16
+		player-anonymous: 51, 51, 16, 16
+
```
Comment (reaperrr) [inline mods/ra/chrome.yaml]: Redundant newline.

File: mods/ra/chrome.yaml [hunk 2]
```diff
@@ -403,32 +411,32 @@ progressbar-thumb:
...
 button-pressed:
 	Inherits: ^Dialog
-	PanelRegion: 640, 0, 1, 1, 126, 126, 1, 1
+	PanelRegion: 641, 1, 2, 2, 122, 122, 2, 2
 
 # 50% grey (disabled button)
 button-disabled:
 	Inherits: ^Dialog
-	PanelRegion: 512, 256, 1, 1, 126, 126, 1, 1
+	PanelRegion: 513, 257, 2, 2, 122, 122, 2, 2
```
Comment (reaperrr) [inline mods/ra/chrome.yaml]: Could you remove those excess tabs while you're at it?

**Comment by pchote** [comment]
Fixed.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/1a78836e37a64b157dee6fe8879deb871d67448a...55f41f46cc75c8b5991cd2421134cfbd2607067a)

**Pull request merged**